tests/unit/store/actions.spec.ts

```diff
@@ -80,6 +86,32 @@ describe( 'actions', () => {
 		} );
 	} );
 
+	describe( 'searchItemValues', () => {
+		it( 'calls the repo and resolves with the result', async () => {
+			const expectedResult = [ { label: 'potato', id: 'Q666', datatype: 'string' } ];
+			const searchProperties = jest.fn().mockResolvedValue(
+				JSON.parse( JSON.stringify( expectedResult ) ),
+			);
+			const searchItemValues = jest.fn().mockResolvedValue(
+				JSON.parse( JSON.stringify( expectedResult ) ),
+			);
+			const actions = createActions(
+				{ searchProperties, searchItemValues },
```

Comment:
You can replace the `searchProperties` argument just with a generic function stub since it is never called anyway. Similarly in the other modified tests in this file.

```suggestion
				{ jest.fn(), searchItemValues },
```